OutOfSchool/Tests/OutOfSchool.WebApi.IntegrationTests/ProviderServiceIntergrationTests/ProviderServiceUpdate.cs

```diff
@@ -0,0 +1,241 @@
+namespace OutOfSchool.WebApi.IntegrationTests.ProviderServiceIntergrationTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using FluentAssertions.Common;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.Extensions.Localization;
+    using Microsoft.Extensions.Logging;
+    using Moq;
+    using NUnit.Framework;
+    using OutOfSchool.Services;
+    using OutOfSchool.Services.Models;
+    using OutOfSchool.Services.Repository;
+    using OutOfSchool.Tests;
+    using OutOfSchool.Tests.Common.TestDataGenerators;
+    using OutOfSchool.WebApi.Models;
+    using OutOfSchool.WebApi.Services;
+    using OutOfSchool.WebApi.Util;
+
+    [TestFixture]
+    public class ProviderServiceUpdate
+    {
+        private const string FakeUserId = "cqQQ876a-BBfb-4e9e-9c78-a0880286ae3c";
+
+        private IProviderService providerService;
+
+        // private Mock<IProviderRepository> providersRepositoryMock;
+        private Mock<IEntityRepository<User>> usersRepositoryMock;
+        private Mock<IRatingService> ratingService;
+        private Mock<IEntityRepository<Address>> adkressRepository;
+        private Mock<IStringLocalizer<SharedResource>> localizer;
+        private Mock<ILogger<ProviderService>> logger;
+
+        private List<Provider> fakeProviders;
+        private User fakeUser;
+
+        private IProviderRepository providerRepository;
+        private Mapper mapper;
+        private DbContextOptions<OutOfSchoolDbContext> unitTestDbOptions;
+
+        private OutOfSchoolDbContext GetContext => new OutOfSchoolDbContext(unitTestDbOptions);
+
+        [SetUp]
+        public async Task SetUp()
+        {
+            unitTestDbOptions = UnitTestHelper.GetUnitTestDbOptions();
+            await using var context = GetContext;
+            fakeProviders = ProvidersGenerator.Generate(10);
```

Comment:
All tests are using `context.Providers.First()`. Why 10 of them should be generated?